### PR TITLE
Downlevel logging

### PR DESCRIPTION
**Pull request by mohitpali**
*Issue #, if available:* https://github.com/awslabs/aws-glue-schema-registry/issues/9

*Description of changes:* Down-leveling logging.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by RushinNaik** [comment]
Looks good.

File: avro-serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/serializers/avro/AWSKafkaAvroSerializer.java [hunk 1]
```diff
@@ -69,7 +69,7 @@ public AWSKafkaAvroSerializer(AwsCredentialsProvider credentialProvider, UUID sc
 
     @Override
     public void configure(@NonNull Map<String, ?> configs, boolean isKey) {
-        log.info("Configuring Amazon Schema Registry Service using these properties: {}", configs);
+        log.info("Configuring Amazon Glue Schema Registry Service using these properties: {}", configs);
```
Comment (blacktooth) [inline avro-serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/serializers/avro/AWSKafkaAvroSerializer.java:72]: Remove Amazon?

File: avro-serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/serializers/avro/AWSKafkaAvroSerializer.java [hunk 2]
```diff
@@ -89,14 +89,14 @@ public void configure(@NonNull Map<String, ?> configs, boolean isKey) {
 
         UUID schemaVersionIdFromRegistry = null;
         if (this.schemaVersionId == null) {
-            log.info("Schema Version Id is null. Trying to register the schema.");
+            log.debug("Schema Version Id is null. Trying to register the schema.");
```
Comment (blacktooth) [inline avro-serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/serializers/avro/AWSKafkaAvroSerializer.java:92]: Are we logging elsewhere that schema is getting registered?

File: common/src/main/java/com/amazonaws/services/schemaregistry/common/configs/GlueSchemaRegistryConfiguration.java [hunk 3]
```diff
@@ -177,7 +177,7 @@ private void validateAndSetCacheTTL(Map<String, ?> configs) {
                 throw new AWSSchemaRegistryException(message, e);
             }
         } else {
-            log.info("Cache Time to live is not found, using default {}", timeToLiveMillis);
+            log.debug("Cache Time to live is not found, using default {}", timeToLiveMillis);
```
Comment (blacktooth) [inline common/src/main/java/com/amazonaws/services/schemaregistry/common/configs/GlueSchemaRegistryConfiguration.java:180]: These are one-time and we should be logging them.

File: common/src/main/java/com/amazonaws/services/schemaregistry/common/configs/GlueSchemaRegistryConfiguration.java [hunk 4]
```diff
@@ -207,7 +207,7 @@ private void validateAndSetTags(Map<String, ?> configs) throws AWSSchemaRegistry
                 throw new AWSSchemaRegistryException(AWSSchemaRegistryConstants.TAGS_CONFIG_NOT_HASHMAP_MSG);
             }
         } else {
-            log.info("Tags value is not defined in the properties. No tags are assigned");
+            log.debug("Tags value is not defined in the properties. No tags are assigned");
```
Comment (blacktooth) [inline common/src/main/java/com/amazonaws/services/schemaregistry/common/configs/GlueSchemaRegistryConfiguration.java:210]: Same.

**Review by blacktooth — COMMENTED** [review]
Reviewed.

File: avro-serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/serializers/avro/AWSKafkaAvroSerializer.java (hunk 2 above)
Comment (mohitpali) [reply avro-serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/serializers/avro/AWSKafkaAvroSerializer.java:92]: Yes

**Pull request closed without merging**